server/manager/src/main/java/org/apache/accumulo/master/MasterClientServiceHandler.java

```diff
@@ -278,7 +278,9 @@ public void shutdownTabletServer(TInfo info, TCredentials c, String tabletServer
 
     log.debug("Seeding FATE op to shutdown " + tabletServer + " with tid " + tid);
 
-    master.fate.seedTransaction(tid, new TraceRepo<>(new ShutdownTServer(doomed, force)), false);
+    master.fate.seedTransaction(tid,
+        new TraceRepo<>(new ShutdownTServer(doomed.getHostAndPort(), doomed.getSession(), force)),
```

Comment:
The constructor can still use this type, since that won't affect serialization. The fields can be derived from the constructor parameter, rather than passed in as separate parameters.

Reply:
Oh yeah, good idea.